### ISSUE TITLE
Inconvenient to add further CSS files

### ISSUE THREAD

**Issue by daniel-beck** [jenkinsci/theme-manager-plugin#26]
`theme-manager` makes it very easy, if following the (implied) structure of an `UnprotectedRootAction` with `getUrlName()` returning `theme-whatever`, to specify the name of a CSS file.

This assumes that a theme only ever specifies a single CSS file. If more than one file is needed/desired, the plugin cannot reuse the existing facility to complete the URL, and instead needs to compute the URL itself.

See e.g. https://github.com/daniel-beck/solarized-theme-plugin/blob/ce7c44a0195193b2fe85bfc0a5748884e775bc1a/src/main/java/io/jenkins/plugins/solarizedtheme/AbstractSolarizedTheme.java#L14-L21

**Comment by timja** [comment]
You can also import the files from the single css file, you don't actually need to add them on the server side.

Might help:
https://github.com/jenkinsci/dark-theme-plugin/blob/master/src/main/resources/io/jenkins/plugins/darktheme/theme-system.css#L1

**Comment by daniel-beck** [comment]
I tried that, but it didn't work with the way I tried to DRY the variable definitions in Solarized Theme Plugin.

Specifically, I have:

* Base definitions: https://github.com/daniel-beck/solarized-theme-plugin/blob/master/src/main/webapp/solarized-base.css
* One of theme theme variants defining variant-dependent variables:
  * https://github.com/daniel-beck/solarized-theme-plugin/blob/master/src/main/webapp/solarized-light.css
  * https://github.com/daniel-beck/solarized-theme-plugin/blob/master/src/main/webapp/solarized-dark.css
  * https://github.com/daniel-beck/solarized-theme-plugin/blob/master/src/main/webapp/solarized-system.css
* The specific definitions for Jenkins variables using the previous two layers: https://github.com/daniel-beck/solarized-theme-plugin/blob/master/src/main/webapp/solarized-definitions.css

Suggestions welcome.

**Comment by timja** [comment]
Add a `getAssetUrl()` method which doesn't have the final suffix allowing you to add additional files?

**Comment by daniel-beck** [comment]
Yes, something similar to https://github.com/daniel-beck/solarized-theme-plugin/blob/ce7c44a0195193b2fe85bfc0a5748884e775bc1a/src/main/java/io/jenkins/plugins/solarizedtheme/AbstractSolarizedTheme.java#L19-L21 should be provided in this plugin.
